documentation/asciidoc/topics/ref_spring_boot_starter_versions.adoc

```diff
@@ -0,0 +1,6 @@
+= Spring Boot Starter Versions
```

Comment:
can we have a community version of this file? 
Just mentioning Spring Boot 2 and to set the `version.infinispan.starter` to 2.1.10.Final
we can copy the Jenkinsfile-release changes from Infinispan to set the correct version.

WDYT?

Reply:
@pruivo made the changes. I wonder if it would be better to use the docs build script for the spring boot starter instead of the jenkins-release file? 

See here: https://github.com/infinispan/infinispan.github.io/blob/aa0b89c0fe64dcf944ebd6d75fd776fe6bc7e0a6/bin/fetch_docs.rb#L161 

Note that we'll have to make these changes for the master branch for upstream. This PR is on the 9.4.x branch. I don't think these docs are published anywhere on the community side, now that I think about it.

Reply:
you're right. we are only publishing the docs from master [1].

[1] https://github.com/infinispan/infinispan.github.io/blob/develop/_config/ispn.yml#L995

It is up to you. we can "ignore" 9.4.x (and 10.1.x!) branch or publish it.


> @pruivo made the changes. I wonder if it would be better to use the docs build script for the spring boot starter instead of the jenkins-release file?
> 
Personally, I would rather change the version in the jenkins file. At least, I would like to see the same HTML when "compiling" locally...